.eslintrc.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
       [
         { pattern: '' },
         { pattern: ' \\(c\\) Copyright IBM Corp. \\d{4}' },
-        { pattern: ' \\(c\\) Copyright Instana Inc. and contributors \\d{4}' },
+        { pattern: '(?: \\(c\\) Copyright Instana Inc. and contributors \\d{4})?' },
```

Comment:
This is actually unrelated to the PR, but this is simply the first PR where this becomes relevant.

With transfer of business from the Instana Inc entity to IBM, the copyright for new files going forward should no longer require this line, so I made it optional.